readme.md

```diff
@@ -182,23 +182,35 @@ input values, and does not produce side effects.
 const greet = (name) => 'Hi, ' + name
 
 greet('Brianne') // 'Hi, Brianne'
-
 ```
 
-As opposed to:
+As opposed to each of the following:
 
 ```js
-
-let greeting
+window.name = 'Brianne'
 
 const greet = () => {
-  greeting = 'Hi, ' + window.name
+  return 'Hi, ' + window.name
```

Comment:
Sorry for the delay, it was pending as I had not hit the review completion button 😁